slurm/datadog_checks/slurm/check.py

```diff
@@ -0,0 +1,442 @@
+# (C) Datadog, Inc. 2024-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+import os
+import subprocess
+from datetime import datetime, timedelta
```

Comment:
Hmm I thought we preferred to use our own internal `time` [module](https://github.com/DataDog/integrations-core/blob/master/datadog_checks_base/datadog_checks/base/utils/time.py) moving forward, for [example](https://github.com/DataDog/integrations-core/blob/master/ibm_mq/datadog_checks/ibm_mq/utils.py).

Reply:
Sounds good. I don't have much of a preference. I swapped the datetime.now with the internal one but I rather still use the datetime module since it has easy way to get total seconds for hours minutes seconds and I didn't want to write my own.